projects/kit/components/input-slider/input-slider.style.less

```diff
@@ -104,18 +104,23 @@
     margin-left: 0.75rem;
 }
 
-.t-slider {
+// TODO: remove wrapper and move all styles to input[type='range'] in v3.0
+.t-slider-wrapper {
     position: absolute;
     top: 100%;
-    left: 0;
+    left: calc(var(--tui-radius-m) / 2);
     right: 0;
-    border-top-left-radius: 0;
-    border-top-right-radius: 0;
     margin: -0.5625rem 0 0;
-    color: transparent;
+}
 
-    :host._disabled &,
-    :host._readonly & {
-        pointer-events: none;
-    }
+// Dont change CSS-selector (we require the such specificity to override default styles without !important)
+input[type='range'] {
```

Comment:
won't that override styles elsewhere? can this break the layout for someone else? or is it encapsulated by angular?

Reply:
> won't that override styles elsewhere?

Component has `ViewEncapsulation.Emulated`, it should not